arkouda-docker/README.md

```diff
@@ -208,4 +245,9 @@ python build_docker_image.py --chapel_version=1.29.0 --image_type=chapel-gasnet-
 python build_docker_image.py --arkouda_tag=v2023.02.08 --chapel_version=1.29.0 --image_type=arkouda-udp-server
 ```
 
+### prometheus-arkouda-exporter
+
+```
+python build_docker_image.py --arkouda_tag=v2023.02.08 --image_type=prometheus-arkouda-exporter
```

Comment:
Just curious here, should we list a later tag. I know in the past we have built the images with a specific tag. Might be worth some discussion about how to keep these up to date without significantly increasing workload.

Reply:
ah, good point, I'll update the README.md. I'll specicy these are example build commands and that the arkouda tag should be the latest

Reply:
@Ethan-DeBandi99 updated the README, please check it out and lemme know what you think.